### ISSUE TITLE
SSH/SDK Mode only working after flash

### ISSUE THREAD

**Issue by eLtMosen** [AsteroidOS/meta-smartwatch#31]
tl:dr;
Below testing is influenced by either choosing USB2 (not working) and USB3 (working) which was not known during taking the logs.

Using the nightly from 2021-12-24 on sparrow.
SDK Mode connection via ssh works right after the flash but fails to connect after reboot.

A: Observations during malfunctioning sdk mode:
- Cable is having good connection, checked using `fastboot devices` multiple times
- There is no "Using SDK Mode" message when plugin in the usb cable
- Spaming `ssh ceres@192.168.2.15` during boot and for a minute after boot does always result in `ssh: connect to host 192.168.2.15 port 22: Network is unreachable`.
- Plug usb cable in and out does not help
- Multiple reboots did not help
- Booting with cable not plugged in, plugging it in some seconds after boot does not help.
- Toggling usb mode to "charge only" and back does not help

B: Even after reflashing to fresh state: (With cable connected during boot)
- right after first boot, with the usb cable still connected from fastboot flash, the SDK Network shows up on my linux computer
- But there is also no connection possible `ssh: connect to host 192.168.2.15 port 22: No route to host`
- Plugging the cable in and out again results in `Network is unreachable` with no network showing up on computer.

### I can reliably get SDK mode to work on first boot after flash now by disconnecting the usb cable after flash and reconnect it after it settled after boot. 
C: During runtime with working SDK Mode:

- It persists ceres restart
- But when plugging the cable in and out, there is no further connection possible `Network is unreachable`
- Then rebooting from settings and booting with usb not connected and letting it settle, sadly does not have the same success but shows behaviour described in A:

D: Edit, tested against 1.0 and found the SDK-Mode to work much more reliable.
- Booting with usb cable plugged in only sometimes leads to A:
- Plugging the cable out and back in always result in working SDK-Mode.

### Edit 2, UGH, this is awkward.
 - Flashing only the 1.0 bootloader image with the 2021-12-24 nightly results in same good performance then when flashing both 1.0 images completely in D:.
 - This happened by accident when not adapting cli commands to renamed image files.
 
 Does that even make technical sense that the booloader image could influence SDK-Mode?
 If not, i will test everything again to double check and wait longer for the device to settle after boot.

**Comment by eLtMosen** [comment]
This is solved at least as of nightlies 2022-03-11.
SDK mode now works reliably like on all other watches.
